src/main/java/com/extrahardmode/ExtraHardMode.java

```diff
@@ -98,16 +97,40 @@ public void onEnable()
         registerModule(AntiFarming.class, new AntiFarming(this));
         registerModule(AnimalCrowdControl.class, new AnimalCrowdControl(this));
         registerModule(AntiGrinder.class, new AntiGrinder(this));
-        registerModule(DebugMode.class, new DebugMode(this));
+        
+
+        boolean ddb = false;
```

Comment:
I think the new lines comes from me copying things over with notepad ++. You know because the first fork got kind of messy. :P